### PR TITLE
Feature/tg/task all

### PR DESCRIPTION
**Pull request by SebastianRonchetti**
General work done, waiting for corrections

### CODE REVIEW

File: src/resources/tasks.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+/* eslint-disable import/no-import-module-exports */
+import express, { json } from 'express';
+
+const fileSystem = require('fs');
```
Comment (JuanMornik) [inline src/resources/tasks.js:4]: Replace with  " import fs from 'fs'; "

File: src/resources/tasks.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+const router = express.Router();
+const jAdress = 'src/data/tasks.json';
+
+// use "require" to import JSON files
+const tasks = require('../data/tasks.json');
```
Comment (JuanMornik) [inline src/resources/tasks.js:10]: Replace this require with import too

File: src/resources/tasks.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  if (comparator) {
+    res.status(200).json({ data: comparator });
+  } else {
+    res.status(404).send('The specified task does not exist');
+  }
+});
+
+router.get('/filterByStatus', (req, res) => {
+  const taskFilter = req.query.status;
+  const compliantTasks = tasks.filter((task) => task.status === taskFilter);
```
Comment (JuanMornik) [inline src/resources/tasks.js:29]: if status is type boolean add toString() method to task.status

**Review by JuanMornik — DISMISSED** [review]

**Review by AlfonsoDalix — APPROVED** [review]
Good work!

**Pull request merged**